src/server/api/controllers/profile.controller.js

```diff
@@ -37,7 +37,31 @@ const getAdminsProfile = async (userId) => {
     return error.message;
   }
 };
+
+const postAdminsProfile = async (adminRole, userId) => {
+  try {
+    const userProfileData = await knex.raw(
+      ` SELECT us.id AS userID, us.name as userName FROM users AS us, user_roles AS ur, roles AS rl WHERE ur.user_id = us.id AND ur.role_id = rl.id AND rl.name = 'student' AND us.id = ? `,
+      [userId],
+    );
+    if (userProfileData[0].length === 0) {
```

Comment:
Did you mean `if (userProfileData.length === 0) {` here?

Reply:
The first element in the array was the actual data. so I meant that.

Reply:
Why not use `await knex().first()`?